tests/validation/tests/v3_api/test_auth.py

```diff
@@ -815,10 +765,16 @@ def create_project_client(request):
     setup["ns2"] = ns2
 
     def fin():
-        client = get_admin_client()
         client.delete(setup["project1"])
         client.delete(setup["project2"])
         delete_cluster_users()
+        if cluster_total == 0:
+            if aws_nodes1 and aws_nodes2:
+                cluster_cleanup(client, cluster1, aws_nodes1)
+                cluster_cleanup(client, cluster2, aws_nodes2)
+        if cluster_total == 1:
+            if aws_nodes1:
+                cluster_cleanup(client, cluster1, aws_nodes1)
```

Comment:
I think this is good, but do we potentially want a way to keep the clusters around?

Reply:
in that case just set `RANCHER_CLEANUP_CLUSTER` to false 👍